From de7cd4d7f6d8525d781f3a9dba3b0b5b654e6272 Mon Sep 17 00:00:00 2001
From: Nathan Scott <nathans@redhat.com>
Date: Mon, 22 Feb 2021 11:32:05 +1100
Subject: [PATCH] Revert "systemd: remove Wants=pmcd from pmlogger and pmie"

This reverts commit e01ab96c813fa31e685ac864031d145f63162088
(for now).

Resolves Red Hat BZs 1929556 and 1930284.

(cherry picked from commit 30ee6b4d6b00b745cb8e44ee6c9ea0a9377398c6)
---
 src/pmie/pmie.service.in         | 1 +
 src/pmlogger/pmlogger.service.in | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/pmie/pmie.service.in b/src/pmie/pmie.service.in
index 75696de40..d234c8a5e 100644
--- a/src/pmie/pmie.service.in
+++ b/src/pmie/pmie.service.in
@@ -4,6 +4,7 @@ Documentation=man:pmie(1)
 After=network-online.target pmcd.service
 Before=pmie_check.timer pmie_daily.timer
 BindsTo=pmie_check.timer pmie_daily.timer
+Wants=pmcd.service
 
 [Service]
 Type=notify
diff --git a/src/pmlogger/pmlogger.service.in b/src/pmlogger/pmlogger.service.in
index f5f374e53..b6273d1a0 100644
--- a/src/pmlogger/pmlogger.service.in
+++ b/src/pmlogger/pmlogger.service.in
@@ -4,6 +4,7 @@ Documentation=man:pmlogger(1)
 After=network-online.target pmcd.service
 Before=pmlogger_check.timer pmlogger_daily.timer pmlogger_daily-poll.timer
 BindsTo=pmlogger_check.timer pmlogger_daily.timer pmlogger_daily-poll.timer
+Wants=pmcd.service
 
 [Service]
 Type=notify
-- 
2.35.3

